src/libs/NextStepUtils.ts

```diff
@@ -97,6 +97,16 @@ function buildNextStep(report: OnyxEntry<Report>, predictedNextStatus: ValueOf<t
     const type: ReportNextStep['type'] = 'neutral';
     let optimisticNextStep: ReportNextStep | null;
 
+    const noActionRequired = {
+        icon: CONST.NEXT_STEP.ICONS.CHECKMARK,
+        type,
+        message: [
+            {
+                text: 'No further action required!',
```

Comment:
@koko57 I think this needs to be translated as per the PR Author Checklist items:

> * [x] I verified any copy / text shown in the product is localized by adding it to src/languages/* files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
  > * [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
  
cc @srikarparsi

Reply:
Thank you for brining this up. I asked for translation confirmation [here](https://expensify.slack.com/archives/C21FRDWCV/p1728520935144059) and will leave an update once we get that.

Reply:
@ikevin127 @srikarparsi we don't translate the other next step actions. The BE also sends untranslated messages for now.

If you decide to translate next steps I think it's better to work on that in another PR

Reply:
That is true, sounds good to tackle this in a future PR